tests/utils.py

```diff
@@ -0,0 +1,86 @@
+import psycopg as pg
+import musicbrainz_bot.config as cfg
+from os import system as system
+from time import perf_counter
+
+"""
+Defines some utility functions for testing on the musicbrainz_test database.
+"""
+
+
+def _get_free_editor_id():
+    """
+    Returns the lowest editor ID that is not currently in use.
+    """
+    with pg.connect(cfg.MB_TEST_DB) as conn:
```

Comment:
ideally the tests would open a single PG connection during setup (I guess similar to `mb_client`) and pass the connection to these functions

Reply:
Hey, can you kindly let me know more about this one?
I thought creating new connections per function would enable better isolation and concurrency for each function. 

How do you recommend creating 1 single session for a test and using it throughout the session? By specifying a fixture in conftest.py maybe? Also, I am curious what benefits does this method bring over the "new conn for every function" approach?

Reply:
> I thought creating new connections per function would enable better isolation and concurrency for each function.

Well, isolation (if needed) is achieved by using transactions.  Do these tests run in parallel?  If so, then separate connections might make sense.  If not, then all commands are running serially anyway.

> How do you recommend creating 1 single session for a test and using it throughout the session? By specifying a fixture in conftest.py maybe?

Yeah, a fixture like you have for the bot client, unless you see a reason that's not workable.

> Also, I am curious what benefits does this method bring over the "new conn for every function" approach?

Mainly to reduce boilerplate: having to specify `with pg.connect(db_URI) as conn:` in every function.

If we had hundreds of tests, it would also be more efficient to reuse a single connection, but that's not much of a concern yet.

Reply:
On it :)

Reply:
Fixed! :D